### PR TITLE
Adds metrics framework documentation

### PR DESCRIPTION
**Pull request by Gaganjuneja**
### Description
_Describe what this change achieves._

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

Closes https://github.com/opensearch-project/documentation-website/issues/6338
### Checklist
- [ x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Gaganjuneja** [comment]
@reta please take a look.

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10141).    
+{: .warning}
+
+
+
+The current OpenSearch stats APIs offer valuable insights into the inner workings of each node and the cluster as a whole. However, they lack certain details such as percentiles and do not provide the semantics of richer metric types like histograms. Consequently, identifying outliers becomes challenging. OpenSearch with Metrics Framework feature adds comprehensive metric support to effectively monitor the cluster.
+
+Developers can easily add new metrics using the Metrics Framework and some of them are already added to visualize the query shaping, see the associated PR [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10724). OpenTelemetry offers decent support for metrics, and we are leveraging the existing `telemetry-otel` plugin to provide the implementation for metrics as well. OpenSearch users can easily consume these metrics through open source OpenTelemetry sinks.
```
Comment (reta) [inline _monitoring-your-cluster/metrics/getting-started.md:16]: I think we don't need to mention other issues / pull requests here, that would be separate features

Comment (reta) [reply _monitoring-your-cluster/metrics/getting-started.md:16]:
```suggestion
Plugin and extension developers should be able to easily add new metrics using the Metrics Framework APIs. By default, the OpenSearch distribution bundles `telemetry-otel` plugin that provide the implementation for metrics instrumentation based on [OpenTelemetry](opentelemetry.io) Java SDK.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+### Enable with Docker containers
+
+If you’re running Docker, add the following line to `docker-compose.yml` under `environment`:
+
+```bash
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true"
+```
+{% include copy.html %}
+
+### Enable for OpenSearch development
```
Comment (reta) [inline _monitoring-your-cluster/metrics/getting-started.md:89]: Do we need this section for user facing documentation?

File: _monitoring-your-cluster/index.md [hunk 3]
```diff
@@ -18,4 +18,5 @@ OpenSearch provides several ways for you to monitor your cluster health and perf
 
 - [Performance analyzer]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/pa/index/) is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
 
-- OpenSearch [Job Scheduler]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/job-scheduler/index/) plugin provides a framework that you can use to build schedules for common cluster management tasks.
\ No newline at end of file
+- OpenSearch [Job Scheduler]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/job-scheduler/index/) plugin provides a framework that you can use to build schedules for common cluster management tasks.
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/index.md:21]:
```suggestion
- The OpenSearch [Job Scheduler]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/job-scheduler/index/) plugin provides a framework that you can use to build schedules for common cluster management tasks.
```

File: _monitoring-your-cluster/index.md [hunk 4]
```diff
@@ -18,4 +18,5 @@ OpenSearch provides several ways for you to monitor your cluster health and perf
...
+- OpenSearch [Metrics Framework]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/metrics/) plugin provides a framework that you can use to export the telemetry metrics to the store of your own choice.
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/index.md:22]:
```suggestion
- The OpenSearch [Metrics Framework]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/metrics/) plugin provides a framework that you can use to export the telemetry metrics to the store of your own choice.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Metrics Framework 
+parent: Trace Analytics
+nav_order: 65
+redirect_from:
+  - /monitoring-your-cluster/metrics/
+---
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10141).
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:9]:
```suggestion

This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10141).    
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+nav_order: 65
+redirect_from:
+  - /monitoring-your-cluster/metrics/
+---
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10141).    
+{: .warning}
+
+
+
+The current OpenSearch stats APIs offer valuable insights into the inner workings of each node and the cluster as a whole. However, they lack certain details such as percentiles and do not provide the semantics of richer metric types like histograms. Consequently, identifying outliers becomes challenging. OpenSearch with Metrics Framework feature adds comprehensive metric support to effectively monitor the cluster.
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:14]:
```suggestion
Whilte the OpenSearch Stats APIs offer insights into the inner workings of each node and an OpenSearch cluster as a whole, the statistics lack certain details, such as percentiles, and do not provide the semantics of richer metric types like histograms. Consequently, identifying outliers within cluster statistic becomes challenging when using only the Stats API. 

The OpenSearch Metric Framework plugin adds comprehensive metric support to effectively monitor an OpenSearch cluster. Using the Metrics Framework APIs, plugin and extension developers can add new monitoring metrics. In addition, the OpenSearch distribution bundles the `telemetry-otel` plugin which provides the implementation for metrics instrumentation based on the [OpenTelemetry](https://opentelemetry.io) Java SDK.
```

File: _monitoring-your-cluster/metrics/getting-started.md (hunk 1 above)
Comment (Naarcha-AWS) [reply _monitoring-your-cluster/metrics/getting-started.md:16]: @reta: I combined this feedback with my suggestion above.

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+Developers can easily add new metrics using the Metrics Framework and some of them are already added to visualize the query shaping, see the associated PR [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10724). OpenTelemetry offers decent support for metrics, and we are leveraging the existing `telemetry-otel` plugin to provide the implementation for metrics as well. OpenSearch users can easily consume these metrics through open source OpenTelemetry sinks.
+
+
+## Get started
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:19]:
```suggestion
## Getting started
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+The Metrics Framework feature is experimental as of OpenSearch 2.11. To begin using the metrics framework feature, you need to first enable the `telemetry feature` using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently using the metrics framework feature flag. It's important to exercise caution when enabling this feature because it can consume system resources. Detailed information on enabling and configuring metrics framework is described in the following sections.
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:21]:
```suggestion
The Metrics Framework feature is experimental as of OpenSearch 2.11. To begin using the metrics framework feature, you need to first enable the `telemetry feature` using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently using the metrics framework feature flag. 

Enabling this feature can consume system resources. Before enabling the Metrics Framework, consider if you have enough cluster resources to allocate.
{: .warning}
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+### Enabling the flag on a node using tarball
+
+The enable flag is toggled using a new Java Virtual Machine (JVM) parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in `config/jvm.options`.
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:25]:
```suggestion
The `enable` flag is toggled using a new Java Virtual Machine (JVM) parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in `config/jvm.options`.
```

File: _monitoring-your-cluster/metrics/getting-started.md (hunk 2 above)
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:89]: I wouldn't recommend it, instead, I would keep this in the GitHub repo with the Metrics Framework source code.

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+    if (numNodes > 1) numberOfNodes = numNodes
+    systemProperty 'opensearch.experimental.feature.telemetry.enabled', 'true'
+    }
+ }
+ ```
+ {% include copy.html %}
+
+ ### Enable metrics framework
+
+Once you've enabled the feature flag, you can enable the metrics framework (which is disabled by default) by using the following setting that enables metrics in the opensearch.yaml:
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:109]:
```suggestion
Once you've enabled the feature flag, you can enable the metrics framework (which is disabled by default) by adding the following setting to `opensearch.yaml`:
```

Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:109]:
```suggestion
Once you've enabled the feature flag, you can enable the Metrics Framework by using the following setting that enables metrics in the opensearch.yaml:
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+Run OpenSearch:
+
+```bash
+./bin/opensearch
+```
+{% include copy.html %}
+
+#### Option 3: Enable from an environment variable
+
+As an alternative to directly modifying `config/jvm.options`, you can define the properties by using an environment variable. You can enable this feature in a single command when you start OpenSearch or by setting an environment variable.
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:63]:
```suggestion
You can enable the Metrics Framework feature in a single command when you start OpenSearch or by setting an environment variable.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+
+ ### Enable metrics framework
+
+Once you've enabled the feature flag, you can enable the metrics framework (which is disabled by default) by using the following setting that enables metrics in the opensearch.yaml:
+
+```bash
+telemetry.feature.metrics.enabled=true
+```
+
+The OpenSearch metrics framework aims to support various telemetry solutions through plugins. The OpenSearch OpenTelemetry plugin `telemetry-otel` is available and must be installed to enable metrics. The following guide provides you with the installation instructions.
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:115]:
```suggestion
The OpenSearch Metrics Frame supports various telemetry solutions through plugins. Use the following instructions to enable the `telemetry-otel` plugin:

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+
+
+### Exporters
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:118]:
```suggestion

```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+
+1. **Publish Interval:** Metrics framework can locally aggregate the metrics for the unique dimensions for the configured publish interval and then export. By default this is configured for 1 minute but can be changed using this cluster setting `telemetry.otel.metrics.publish.interval`
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:120]:
```suggestion
1. **Publish Interval:** The Metrics Framework can locally aggregate the metrics with unique dimensions about the configured publish interval, and then export those metrics. By default, the interval is 1 minute. However, you be change the interval using the `telemetry.otel.metrics.publish.interval` cluster setting.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+2. **Exporters:** Exporters are responsible for persisting the data. OpenTelemetry provides several out-of-the-box exporters, and OpenSearch supports the following:
```
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:121]:
```suggestion
2. **Exporters:** Exporters are responsible for persisting the data. OpenTelemetry provides several out-of-the-box exporters. OpenSearch supports the following exporters:
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

**Comment by Gaganjuneja** [comment]
@Naarcha-AWS and @reta thanks for your help here.

File: _monitoring-your-cluster/index.md (hunk 3 above)
Comment (natebower) [inline _monitoring-your-cluster/index.md:21]:
```suggestion
- The OpenSearch [Job Scheduler]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/job-scheduler/index/) plugin provides a framework that you can use to build schedules for common cluster management tasks.
```

File: _monitoring-your-cluster/index.md [hunk 16]
```diff
@@ -18,4 +18,5 @@ OpenSearch provides several ways for you to monitor your cluster health and perf
 
 - [Performance analyzer]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/pa/index/) is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
 
-- OpenSearch [Job Scheduler]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/job-scheduler/index/) plugin provides a framework that you can use to build schedules for common cluster management tasks.
\ No newline at end of file
+- OpenSearch [Job Scheduler]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/job-scheduler/index/) plugin provides a framework that you can use to build schedules for common cluster management tasks.
+- The OpenSearch [Metrics Framework]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/metrics/) plugin provides a framework that you can use to export the telemetry metrics to the store of your own choice.
```
Comment (natebower) [inline _monitoring-your-cluster/index.md:22]:
```suggestion
- The OpenSearch [Metrics Framework]({{site.url}}{{site.baseurl}}/monitoring-your-cluster/metrics/) plugin provides a framework that you can use to export the telemetry metrics to the store of your choice.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+parent: Trace Analytics
+nav_order: 65
+redirect_from:
+  - /monitoring-your-cluster/metrics/
+---
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10141).    
+{: .warning}
+
+While the OpenSearch Stats APIs offer insights into the inner workings of each node and an OpenSearch cluster as a whole, the statistics lack certain details, such as percentiles, and do not provide the semantics of richer metric types like histograms. Consequently, identifying outliers within cluster statistic becomes challenging when using only the Stats API.
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:13]:
```suggestion
While the OpenSearch Stats APIs offer insight into the inner workings of each node and an OpenSearch cluster as a whole, the statistics lack certain details, such as percentiles, and do not provide the semantics of richer metric types, like histograms. Consequently, identifying outliers within cluster statistics becomes challenging when using only the Stats API. 
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+redirect_from:
+  - /monitoring-your-cluster/metrics/
+---
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10141).    
+{: .warning}
+
+While the OpenSearch Stats APIs offer insights into the inner workings of each node and an OpenSearch cluster as a whole, the statistics lack certain details, such as percentiles, and do not provide the semantics of richer metric types like histograms. Consequently, identifying outliers within cluster statistic becomes challenging when using only the Stats API. 
+
+The metrics framework feature adds comprehensive metric support to effectively monitor an OpenSearch cluster. Using the Metrics Framework APIs, plugin and extension developers can add new monitoring metrics. In addition, the OpenSearch distribution bundles the `telemetry-otel` plugin which provides the implementation for metrics instrumentation based on the [OpenTelemetry](https://opentelemetry.io) Java SDK.
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:15]:
```suggestion
The metrics framework feature adds comprehensive metrics support to effectively monitor an OpenSearch cluster. Using the Metrics Framework APIs, plugin, and extension, developers can add new monitoring metrics. In addition, the OpenSearch distribution bundles the `telemetry-otel` plugin, which provides the implementation for metrics instrumentation based on the [OpenTelemetry](https://opentelemetry.io) Java SDK.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+
+
+## Getting started
+
+The Metrics Framework feature is experimental as of OpenSearch 2.11. To begin using the metrics framework feature, you need to first enable the `telemetry feature` using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently using the metrics framework feature flag.
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:20]:
```suggestion
The metrics framework feature is experimental as of OpenSearch 2.11. To begin using the metrics framework feature, you need to first enable the `telemetry feature` by using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently by using the metrics framework feature flag. 
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+
+The metrics framework feature adds comprehensive metric support to effectively monitor an OpenSearch cluster. Using the Metrics Framework APIs, plugin and extension developers can add new monitoring metrics. In addition, the OpenSearch distribution bundles the `telemetry-otel` plugin which provides the implementation for metrics instrumentation based on the [OpenTelemetry](https://opentelemetry.io) Java SDK.
+
+
+## Getting started
+
+The Metrics Framework feature is experimental as of OpenSearch 2.11. To begin using the metrics framework feature, you need to first enable the `telemetry feature` using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently using the metrics framework feature flag. 
+
+Enabling this feature can consume system resources. Before enabling the metrics framework, consider if you have enough cluster resources to allocate.
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:22]:
```suggestion
Enabling this feature can consume system resources. Before enabling the metrics framework feature, determine whether you have sufficient cluster resources to allocate.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 21]
```diff
@@ -0,0 +1,97 @@
...
+{: .warning}
+
+### Enabling the feature flag on a node using tarball
+
+The `enable` flag is toggled using a new Java Virtual Machine (JVM) parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in `config/jvm.options`.
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:27]: Will "new" quickly become outdated? If so, I would remove.

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+
+#### Option 1: Enable the experimental feature flag in the `opensearch.yml` file
+
+1. Navigate to your OpenSearch directory using the following command:
+
+  ```bash
+  cd \path\to\opensearch
+  ```
+
+2. Open your the `opensearch.yaml` file.
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:37]:
```suggestion
2. Open your `opensearch.yaml` file.
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 23]
```diff
@@ -0,0 +1,97 @@
...
+  ```bash
+  opensearch.experimental.feature.telemetry.enabled=true
+  ```
+  {% include copy.html %}
+
+4. Save your changes and close the file.
+
+#### Option 2: Modify jvm.options
+
+To enable the metrics framework feature using `jvm`, add the following line to `config/jvm.options` before starting the OpenSearch:
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:49]:
```suggestion
To enable the metrics framework feature using `jvm`, add the following line to `config/jvm.options` before starting OpenSearch:
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 24]
```diff
@@ -0,0 +1,97 @@
...
+
+```bash
+-Dopensearch.experimental.feature.telemetry.enabled=true
+```
+{% include copy.html %}
+
+#### Option 3: Enable from an environment variable
+
+You can enable the metrics framework in a single command by adding the metrics framework environment variable to the `OPENSEARCH_JAVA_OPTS` command, as shown in the following example:
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:58]:
```suggestion
You can enable the metrics framework feature with a single command by adding the metrics framework environment variable to the `OPENSEARCH_JAVA_OPTS` command, as shown in the following example:
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 25]
```diff
@@ -0,0 +1,97 @@
...
+
+You can also define the environment variable separately before running OpenSearch by running the following command:
+
+```bash
+export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true"
+ ./bin/opensearch
+```
+{% include copy.html %}
+
+### Enable with Docker containers
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:73]: "Enable with a Docker container"?

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 26]
```diff
@@ -0,0 +1,97 @@
...
+
+If you’re running OpenSearch using Docker, add the following line to `docker-compose.yml` under `environment`:
+
+```bash
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.telemetry.enabled=true"
+```
+{% include copy.html %}
+
+
+### Enable metrics framework
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:83]:
```suggestion
### Enable the metrics framework feature
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 27]
```diff
@@ -0,0 +1,97 @@
...
+
+Once you've enabled the feature flag, you can enable the metrics framework by using the following setting that enables metrics in the opensearch.yaml:
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:85]:
```suggestion
Once you've enabled the feature flag, you can enable the metrics framework feature by using the following setting, which enables metrics in the `opensearch.yaml` file:
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 28]
```diff
@@ -0,0 +1,97 @@
...
+
+```bash
+telemetry.feature.metrics.enabled=true
+```
+
+The metrics framework supports various telemetry solutions through plugins. Use the following instructions to enable the `telemetry-otel` plugin:
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:91]:
```suggestion
The metrics framework feature supports various telemetry solutions through plugins. Use the following instructions to enable the `telemetry-otel` plugin:
```

File: _monitoring-your-cluster/metrics/getting-started.md [hunk 29]
```diff
@@ -0,0 +1,97 @@
...
+
+
+1. **Publish Interval:** The Metrics Framework can locally aggregate the metrics with unique dimensions about the configured publish interval, and then export those metrics. By default, the interval is 1 minute. However, you be change the interval using the `telemetry.otel.metrics.publish.interval` cluster setting.
```
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:94]:
```suggestion
1. **Publish interval:** The metrics framework feature can locally aggregate metrics with unique dimensions about the configured publish interval and then export those metrics. By default, the interval is 1 minute. However, you can change the interval using the `telemetry.otel.metrics.publish.interval` cluster setting.
```

Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:94]: "dimensions" reads as the wrong word here. Do we mean "information"?

File: _monitoring-your-cluster/metrics/getting-started.md (hunk 28 above)
Comment (natebower) [inline _monitoring-your-cluster/metrics/getting-started.md:91]: The two items below do not read as "instructions" (we don't tell the reader to do anything).

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS @Gaganjuneja Please see my comments and changes and let me know if you have any questions. Thanks!

File: _monitoring-your-cluster/metrics/getting-started.md (hunk 21 above)
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:27]:
```suggestion
The `enable` flag is toggled using a Java Virtual Machine (JVM) parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in `config/jvm.options`.
```

File: _monitoring-your-cluster/metrics/getting-started.md (hunk 25 above)
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/metrics/getting-started.md:73]:
```suggestion
### Enable with Docker 
```

**Pull request merged**